perun-core/src/test/java/cz/metacentrum/perun/core/entry/AttributesManagerEntryIntegrationTest.java

```diff
@@ -8197,11 +8197,12 @@ public void getNamespaceFromAttributeName() throws Exception {
 public void getFriendlyNameFromAttributeName() throws Exception {
 	System.out.println("attributesManager.getFriendlyNameFromAttributeName");
 
-	String nameSpace = attributesManager.getFriendlyNameFromAttributeName("urn:perun:facility:attribute-def:opt:attr_manager_test_attribute");
-	assertTrue("get wrong namespace from name",nameSpace.equals("attr_manager_test_attribute"));
+	String nameSpace = attributesManager.getFriendlyNameFromAttributeName("urn:perun:facility:attribute-def:opt:attr-manager-test-attribute");
+	assertTrue("get wrong namespace from name",nameSpace.equals("attr-manager-test-attribute"));
 
 }
 
+@Ignore
```

Comment:
Use friendly name login-namespace:META-login for this purpose.
There are really one extra ":" and that's bad friendlyName of attribute.